src/bci_build/package/appcollection.py

```diff
@@ -0,0 +1,122 @@
+"""Application Collection Containers that are generated with the BCI tooling"""
+
+from dataclasses import dataclass
+
+from bci_build.package import CAN_BE_LATEST_OS_VERSION
+from bci_build.package import DOCKERFILE_RUN
+from bci_build.package import _SUPPORTED_UNTIL_SLE
+from bci_build.package import ApplicationStackContainer
+from bci_build.package import Arch
+from bci_build.package import OsVersion
+from bci_build.package import ParseVersion
+from bci_build.package import Replacement
+
+
+@dataclass
+class ApplicationCollectionContainer(ApplicationStackContainer):
+    """Containers for the Rancher Application Collection Distribution Platform."""
+
+    @property
+    def _registry_prefix(self) -> str:
+        if self.is_opensuse:
+            return super()._registry_prefix
+        return "containers"
+
+    @property
+    def registry(self) -> str:
+        if self.is_opensuse:
+            return super().registry
+        return "dp.apps.rancher.io"
+
+    @property
+    def url(self) -> str:
+        if self.is_opensuse:
+            return super().url
+        return f"https://apps.rancher.io/applications/{self.name}"
+
+    @property
+    def title(self) -> str:
+        if self.is_opensuse:
+            return super().title
+        return self.pretty_name
+
+    @property
+    def eula(self) -> str:
+        if self.is_opensuse:
+            return "sle-bci"
+        return "sle-eula"
+
+    def __post_init__(self) -> None:
+        super().__post_init__()
+        if not self.is_opensuse and not self.exclusive_arch:
+            self.exclusive_arch = [Arch.AARCH64, Arch.X86_64]
+
+
+_TOMCAT_VERSIONS: list[int] = [9, 10]
+assert _TOMCAT_VERSIONS == sorted(_TOMCAT_VERSIONS)
+
+TOMCAT_CONTAINERS = [
+    ApplicationCollectionContainer(
```

Comment:
In my understanding, we should publish to the SUSE registry and as a Rancher application. With this change, it seems only Rancher is supported.

Am I missing something? Won't this affect current images?

Reply:
we have never published the current images, this was just dead code before.

Reply:
I missed that. Not an issue anymore.